installer/resources/pacbot_app/files/DB.sql

```diff
@@ -1778,6 +1786,11 @@ INSERT IGNORE INTO cf_Policy (`policyId`,`policyName`,`policyDesc`,`resolution`,
 
 INSERT IGNORE INTO cf_Policy (`policyId`,`policyName`,`policyDesc`,`resolution`,`policyUrl`,`policyVersion`,`status`,`userId`,`createdDate`,`modifiedDate`,`resolutionUrl`) VALUES ('Disable_kubernetes_DashBoard','Disable kubernetes DashBoard','Disable the Kubernetes dashboard to prevent the need to manage its individual access interface and to eliminating it as an attack vector.',NULL,NULL,'version-1',NULL,NULL,'2022-11-22','2022-11-22','https://github.com/PaladinCloud/CE/wiki/Azure-Policy');
 
+
+INSERT IGNORE INTO cf_Policy (`policyId`,`policyName`,`policyDesc`,`resolution`,`policyUrl`,`policyVersion`,`status`,`userId`,`createdDate`,`modifiedDate`,`resolutionUrl`) VALUES ('GCP_Cloud_Function_configured_with_overly_permissive_Ingress_setting','GCP_Cloud_Function_configured_with_overly_permissive_Ingress_setting','GCP Cloud Function configured with overly permissive Ingress setting',NULL,NULL,'version-1',NULL,NULL,'2022-11-22','2022-11-22','https://github.com/PaloAltoNetworks/prisma-cloud-policies/blob/master/policies/GCP-Cloud-Function-configured-with-overly-permissive-Ingress-setting.json');
+INSERT IGNORE INTO cf_Policy (`policyId`,`policyName`,`policyDesc`,`resolution`,`policyUrl`,`policyVersion`,`status`,`userId`,`createdDate`,`modifiedDate`,`resolutionUrl`) VALUES ('GCP_Cloud_Function_not_enabled_with_VPC_connector','GCP_Cloud_Function_not_enabled_with_VPC_connector','GCP Cloud Function not enabled with VPC connector',NULL,NULL,'version-1',NULL,NULL,'2023-01-18','2023-01-18','https://github.com/PaloAltoNetworks/prisma-cloud-policies/blob/master/policies/GCP-Cloud-Function-not-enabled-with-VPC-connector.json');
+INSERT IGNORE INTO cf_Policy (`policyId`,`policyName`,`policyDesc`,`resolution`,`policyUrl`,`policyVersion`,`status`,`userId`,`createdDate`,`modifiedDate`,`resolutionUrl`) VALUES ('Enable_Https_For_Cloudfunc','Enable_Https_For_Cloudfunc','Enable https for Cloud Functions',NULL,NULL,'version-1',NULL,NULL,'2023-01-18','2023-01-18','https://github.com/PaloAltoNetworks/prisma-cloud-policies/blob/master/policies/GCP-Cloud-Function-HTTP-trigger-is-not-secured.json');
+
```

Comment:
instead of this now we need to add entries in the new cf_PolicyTable. Please refer DB_Policy.sql